serverlessworkflow/modules/ROOT/nav.adoc

```diff
@@ -1,59 +1,59 @@
 // * xref:index.adoc[Home]
 * Getting Started
-** xref:getting-started/create-your-first-workflow-service.adoc[Creating your first Serverless Workflow service]
+** xref:getting-started/create-your-first-workflow-service.adoc[Creating your first workflow service]
 ** xref:getting-started/cncf-serverless-workflow-specification-support.adoc[CNCF Serverless Workflow specification]
-** xref:getting-started/getting-familiar-with-our-tooling.adoc[Getting familiar with Serverless Workflow tooling]
+** xref:getting-started/getting-familiar-with-our-tooling.adoc[Getting familiar with {context} tooling]
 * Core
-** xref:core/understanding-jq-expressions.adoc[jq expressions in Serverless Workflow]
-** xref:core/understanding-workflow-error-handling.adoc[Error handling in Serverless Workflow]
-** xref:core/working-with-parallelism.adoc[Parallelism in Serverless Workflow]
-** xref:core/configuration-properties.adoc[Configuration properties in Serverless Workflow]
+** xref:core/understanding-jq-expressions.adoc[jq expressions in {context}]
+** xref:core/understanding-workflow-error-handling.adoc[Error handling in {context}]
+** xref:core/working-with-parallelism.adoc[Parallelism in {context}]
+** xref:core/configuration-properties.adoc[Configuration properties in {context}]
 //** xref:core/accessing-workflow-metainformation-in-runtime.adoc[Accessing workflow metainformation in runtime]
 //** xref:core/defining-an-input-schema-for-workflows.adoc[Defining an input schema for your workflows]
-** xref:core/custom-functions-support.adoc[Custom functions for your Serverless Workflow service]
+** xref:core/custom-functions-support.adoc[Custom functions for your {context} service]
 * Tooling
 ** xref:tooling/serverless-workflow-editor/swf-editor-overview.adoc[Serverless Workflow editor]
 ** xref:tooling/serverless-workflow-editor/swf-editor-vscode-extension.adoc[VS Code extension for Serverless Workflow editor]
 ** xref:tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc[Chrome GitHub extension for Serverless Workflow editor]
 ** xref:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-overview.adoc[Kogito Serverless Workflow Tools extension in Quarkus Dev UI]
 *** xref:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-instances-page.adoc[Workflow Instances in Kogito Serverless Workflow Tools extension]
 *** xref:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc[Workflow Definitions in Kogito Serverless Workflow Tools extension]
```

Comment:
These will always refer to Serverless Workflow?

Reply:
Yes, since the extension name is Kogito Serverless Workflow Tools extension